.platform.app.yaml

```diff
@@ -77,12 +77,10 @@ mounts:
 # The hooks that will be performed when the package is deployed.
 hooks:
   build: |
-    vendor/bin/typo3cms install:generatepackagestates --activate-default=true
-    cd web
-    touch FIRST_INSTALL
-    cd typo3conf
-    ln -sf writeable/LocalConfiguration.php LocalConfiguration.php
-    ln -sf writeable/ENABLE_INSTALL_TOOL ENABLE_INSTALL_TOOL
+    vendor/bin/typo3cms install:generatepackagestates
```

Comment:
This will complain on a fresh install. Why would we not want to install the default packages?